Base/usr/share/man/man6/Minesweeper.md

```diff
@@ -0,0 +1,17 @@
+## Name
+
+Minesweeper
+
+## Synopsis
+
+```**sh
+$ Minesweeper
+```
+
+## Description
+
+The goal is to find all the mines without detonating them.
+
+A player reveals what is underneath a tile by clicking on it. If it is a mine underneath the player loses. Otherwise the tile will show how many neighbouring mines there are to the tile. If no neighbouring mines then all the neighbouring tiles are revealed recursively.
```

Comment:
```suggestion
The player reveals what is underneath a tile by clicking on it. If there is a mine underneath, the player loses. Otherwise the tile will show how many neighbouring mines there are to the tile. If there are no neighboring mines, all the neighboring tiles are revealed recursively.
```